README.md

```diff
@@ -1,104 +1,72 @@
-# Kernel Containers
+<p align="center">
+  <img src="static/images/Kernel-Wordmark_Accent.svg" alt="Kernel Logo" width="55%">
+  <br/><br/>
+  Browsers, nicely packaged in containers
```

Comment:
Technically speaking a unikernel isn't a container, so maybe "nicely packaged in containers or unikernels"? Or maybe just "Browsers, nicely packaged"?